src/main/java/nl/tudelft/dnainator/ui/drawables/phylogeny/InternalNode.java

```diff
@@ -0,0 +1,62 @@
+package nl.tudelft.dnainator.ui.drawables.phylogeny;
+
+import javafx.beans.binding.Bindings;
+import javafx.beans.binding.BooleanBinding;
+
+import java.util.List;
+import java.util.stream.Collectors;
+
+/**
+ * This class represents internal nodes in the phylogenetic tree. Each internal node has a list
+ * of children (even though in our case the tree is always binary). It binds its own inactive property
+ * to the AND of its children's inactive properties to automatically update itself.
+ */
+public class InternalNode extends AbstractNode {
+	private List<AbstractNode> children;
+
+	/**
+	 * Constructs a new {@link InternalNode}.
+	 * @param children The children that form the subtree of which this {@link InternalNode} is
+	 *                 the root node.
+	 * @param x This node's center x-coordinate.
+	 * @param y This node's center y-coordinate.
+	 * @param dim This node's dimensions.
+	 */
+	public InternalNode(List<AbstractNode> children, double x, double y, double dim) {
+		super(x, y, dim);
+		this.children = children;
+//		inactiveProperty().bind(children.stream()
+//				.collect(Collectors.reducing(, child -> child.inactiveProperty(), Bindings::and)));
```

Comment:
Maybe make two instances for left and right subtree, since it has been confirmed that the tree is always binary.
Either that, or make it work for n-ary trees.